src/views/admin/PersonCreationView.vue

```diff
@@ -101,17 +130,25 @@
         vorname: selectedVorname.value as string,
       },
     };
-    await personStore.createPerson(unpersistedPerson);
-    resetForm();
+    personStore.createPerson(unpersistedPerson).then(async (personResponse: PersonendatensatzResponse) => {
```

Comment:
Hier nutzen wir erstmal den vorherigen Endpunkt, der nur Vorname/Nachname braucht und dann mit der Id dieser Person machen wir einen zweiten Request um den Kontext zu erstellen. 

Das Problem wäre, wenn der zweite Endpunkt einen Fehler wirft, dann wird trotzdem der erste Endpunkt die Person anlegen, nur ohne Kontext... Und in der Ergebnisliste wird die Person ohne Rolle/ Organisationsebene angezeigt und davor auch eine Alert, weil ja personStore.errorCode present ist.

Reply:
Würde Ich sagen ja.. um eine Person anzulegen müssen nicht 2 Endpunkte beteiligt sein... Eine Person ohne Kontext macht auch keinen Sinn (Wenn der zweite Requests failed).